packages/charting/src/components/StackedBarChart/StackedBarChart.base.tsx

```diff
@@ -127,12 +153,39 @@ export class StackedBarChartBase extends React.Component<IStackedBarChartProps,
     });
   }
 
+  private _getTotalChartData(data: IChartProps): number {
+    let sum = 0;
+    if (data.chartData && data.chartData.length > 0) {
+      const chartData = data.chartData;
+      for (let i = 0, n = chartData.length; i < n; i++) {
+        sum += chartData[i].data || 0;
+      }
+    }
+
+    return sum;
+  }
+
+  private _createBenchmark(totalData: number, dataPoint: IChartDataPoint): JSX.Element {
+    const ratio = (dataPoint.data! / totalData) * 100;
+
+    const styles = {
```

Comment:
move this into the styles file. To pass the info add stylesProps. This way if someone needs to change the border style or anything else they can do so without trouble. RIght now this would be really hard to customize.

Reply:
In this way we can also change the border style by pass the customize styles to StackedBarChart component. So it can be customized.

There are two ways to move it to the styles file.
1. Add separate styles for benchmark and target triangles.
We need to add **benchmarkTriangle** and **targetTriangle** properties in **IStackedBarChartStyles**. This way will introduce **duplicated code** because many styles of benchmarkTriangle and targetTriangle are the same.
2. reuse triangle style for benchmark and target 
We need to calling our style function twice to pass the info of benchmark styles and target styles. This is going to be **bad for perf**. 

I can't find a good way to move this into the styles file and I think this code is fine.